server/methods/checkUserPassword.js

```diff
@@ -0,0 +1,9 @@
+import { Meteor } from 'meteor/meteor';
+
+import { passwordPolicy } from '../../app/lib';
+
+Meteor.methods({
+	checkUserPassword(password) {
+		return passwordPolicy.validate(password);
```

Comment:
IMHO we should execute the passwordPolicy at client side, we just need to make the settings it depends on public